src/main/java/net/minestom/server/instance/Instance.java

```diff
@@ -405,7 +405,7 @@ public void setTimeUpdate(@Nullable Duration timeUpdate) {
      * @return the {@link TimeUpdatePacket} with this instance data
      */
     private @NotNull TimeUpdatePacket createTimePacket() {
-        return new TimeUpdatePacket(worldAge, time);
+        return new TimeUpdatePacket(worldAge, timeRate == 0 ? -time : time);
```

Comment:
Maybe do `-Math.abs(time)` because projects probably did this previously with negative time

Reply:
> Maybe do `-Math.abs(time)` because projects probably did this previously with negative time

Alright, modifying